File: Domain/Models/Event.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Domain.Models
{
    public class Event
    {
        public int ID { get; set; }
        public int HostedByUserID { get; set; }
        public string Name { get; set; }
        public string Location { get; set; }
        public DateTime Date { get; set; }
        public string Description { get; set; }
        public int Type { get; set; }
        public bool IsActive { get; set; }
        public bool IsPublic { get; set; }
        public bool IsFree { get; set; }
        public bool IsDonationRequired { get; set; }
    }

}

File: WWWRubberCityFoundation/src/app/models/event.ts
export class Event {
    id: number;
    hostedByUserId: number;
    name: string;
    location: string;
    date: Date;
    description: string;
    type: number;
    isActive: boolean;
    isPublic: boolean;
    isFree: boolean;
    isDonationRequired: boolean;
  
    constructor(
      id: number,
      hostedByUserId: number,
      name: string,
      location: string,
      date: Date,
      description: string,
      type: number,
      isActive: boolean,
      isPublic: boolean,
      isFree: boolean,
      isDonationRequired: boolean
    ) {
      this.id = id;
      this.hostedByUserId = hostedByUserId;
      this.name = name;
      this.location = location;
      this.date = date;
      this.description = description;
      this.type = type;
      this.isActive = isActive;
      this.isPublic = isPublic;
      this.isFree = isFree;
      this.isDonationRequired = isDonationRequired;
    }
  }
  